secure_message/resources/messages.py

```diff
@@ -145,18 +146,19 @@ def _inform_case_service(message):
 
     @staticmethod
     def _get_user_name(user, message):
-        user_name = 'Unknown user'
+
         if message.msg_from == constants.BRES_USER:
-            user_name = constants.BRES_USER
-        else:
-            user_data = internal_user_service.get_user_details(message.msg_from) if user.is_internal else party.get_user_details(message.msg_from)
-
-            if user_data:
-                first_name = user_data.get('firstName', '')
-                last_name = user_data.get('lastName', '')
-                full_name = f'{first_name} {last_name}'.strip()
-                if full_name:
-                    user_name = full_name
+            return constants.BRES_USER
+
+        user_name = 'Unknown user'
```

Comment:
Will we ever not have a username?  If we don't, what happens?  Hardcoding an 'unknown user' name seems odd to me but I explain exactly why...